docs/whatsnew/0.9.0.rst

```diff
@@ -0,0 +1,2 @@
+.. changelog::
+   :towncrier: ../../
```

Comment:
```suggestion
.. changelog::
   :towncrier: ../../
   :towncrier-skip-if-empty:
```

If we use the [`:towncrier-skip-if-empty:` option](https://sphinx-changelog.readthedocs.io/en/latest/#skipping-towncrier-on-release-builds), then we could just name this file `dev.rst` and not have to worry about renaming/deleting it during releases.

Reply:
Cool!

Reply:
Readying a little more about this, this only works if we use the `:changelog-file:` option.  We still get the an error since we're include a blank file (i.e. a file without a heading) in the `toctree` in `docs/whatsnew/index.rst`.

My suggestion, we keep this file as `dev.rst` with the following content...

```rst
:orphan:

.. changelog::
   :towncrier: ../../

```

And we just delete the `dev` line in `docs/whatsnew/index.rst` during a release and add it back after the release.  We would not need to do anything with `dev.rst` because we've marked it as `:orphan:`.

Reply:
✅